packages/gcf-utils/src/configuration.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+export const RUNNING_IN_TEST = process.env.NODE_ENV === 'test';
+// Adding 30 second delay for each batch with 30 tasks
+export const DEFAULT_FLOW_CONTROL_DELAY_IN_SECOND = 30;
+
+export interface WrapConfig {
```

Comment:
Please add a comment for this interface.

Reply:
Done